.github/workflows/CI.yml

```diff
@@ -704,6 +704,52 @@ jobs:
           cd testing-framework/terraform
           make terraformCleanup
 
+  run-logs-testbed:
+    runs-on: ubuntu-latest
+    needs: [ e2etest-preparation, create-test-ref]
```

Comment:
```suggestion
    needs: [ e2etest-release, create-test-ref]
```

Reply:
Don't you need the image to be published for this test to be executed? that happens in `e2etest-release`. So I think you will need to add that as an extra `need` here.

Reply:
Makes sense